### PR TITLE
https://issues.redhat.com/browse/ACM-6423 Add API

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.8 and 2.7.

Please review as I'm not familiar with the APIs, but from what I could tell there are no required spec fields in this case. If there are, let me know which ones we would like to add and what they do.

### CODE REVIEW

File: apis/managedclusterset.json.adoc [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+|===
+|Name|Description|Schema
+|*apiVersion* +
+__required__|Versioned schema of the `ManagedClusterSet`.|string
+|*kind* +
+__required__|String value that represents the REST resource.|string
+|*metadata* +
+__required__|Metadata of the `ManagedClusterSet`.|object
+|*spec* +
+__required__|Specification of the `ManagedClusterSet`.|<<_rhacm-docs_apis_clustersetbinding_jsonclustersetbinding_spec,spec>>
```
Comment (brian-jarvis) [inline apis/managedclusterset.json.adoc]: This creates a link to the spec, but I do not see the spec in the file.

Comment (oafischer) [reply apis/managedclusterset.json.adoc]: Fixed, thanks for catching that!

Comment (brian-jarvis) [reply apis/managedclusterset.json.adoc]: @oafischer  the fix was to remove the link?  Why aren't we documenting the spec?

Comment (oafischer) [reply apis/managedclusterset.json.adoc]: Sorry, I'm probably missing something and if so let me know, but I don't see anything that needs to be documented, based on how we treat other spec fields in our existing documentation.

**Review by brian-jarvis — APPROVED** [review]

**Review by dockerymick — APPROVED** [review]
I noticed that ManagedClusterSet is written a couple of ways. I appreciate that the `kind` name matches when there is a hyperlink. We can improve the name consistency as tech debt for 2.9. Let me know your thoughts.

**Comment by oafischer** [comment]
Looks like a copy/paste error, I'll fix it!

**Comment by oafischer** [comment]
@dockerymick I just checked, all of our API docs have the same inconsistencies. I think it's to avoid lower case in the title and main paragraph. My suggestion is to keep the version I had at the time of review and eventually fix all the API docs to make them consistent.

**Pull request merged**